bcipy/signal_model/mach_learning/m_estimator/m_estimator.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import scipy as sc
+
+
+def eigsorted(cov):
+    """
+    Find and sort eigen values and vectors
+    :param cov: covariance matrix
+    :return: sorted eigen values and vectors
+    """
+
+    vals, vecs = np.linalg.eigh(cov)
+    order = vals.argsort()[::-1]
+    return vals[order], vecs[:, order]
+
+
+def u_func(t, b, c_square):
+    """
+    Huber's loss function
+    :param t: Mahalanobis distance
+    :param b: a constant in p
+    :param c_square: a constant in p and q
+    :return: weight of sample
+    """
+
+    if t <= c_square:
+        return 1./b
+    else:
+        return c_square/(t*b)
+
+
+def mean_update(X, mean, sigma_inv, b, c_square):
```

Comment:
lower case variable names. use explicit names... so `data` etc

Reply:
Tab, some of the namings you are opposing are fundamentally used in many applications and definitions in papers. Capital letters are used of matrices and capital X is generally used for design matrix, i.e. data matrix. In the link below look at fit method, there is a parameter X for data matrix and y for label vector:

http://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html

Here is the general definiton of design matrix:
https://en.wikipedia.org/wiki/Design_matrix

I don't want to change these, since I think these are the correct namings for the parameter that is passed.

Reply:
I would not rely on what other libraries do to justify your naming. Other than a capital X to denote data, their variable naming is actually PEP complaint. We use snake case (an issue I mention besides naming throughout this pull request) and, with very few exceptions, no single letter variable names. Here is a nice guide: https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming

Reply:
Okay. I'm changing it.